api/src/main/java/com/epam/pipeline/manager/docker/DockerContainerOperationManager.java

```diff
@@ -72,9 +72,9 @@
     private static final String GLOBAL_KNOWN_HOSTS_FILE = "GlobalKnownHostsFile=/dev/null";
     private static final String USER_KNOWN_HOSTS_FILE = "UserKnownHostsFile=/dev/null";
     private static final String COMMIT_COMMAND_TEMPLATE = "curl -k -s \"%s\" | sudo -E /bin/bash " +
-            "--login /dev/stdin %s %s %s %s %d %s %b %b %d %s %d %d %s %s %s %b &> ~/commit_pipeline.log &";
+            "--login /dev/stdin %s %s %s %s %d %s %b %b %d %s %d %d %s %s %s %b %s %s &> ~/commit_pipeline.log &";
```

Comment:
I think that we should find a better way of building commit commands, with this number of arguments it is unreadable.